src/site/content/en/sandbox/contributor-profile/index.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+noindex: true
+title: Contributor profile
+authors:
+  - mfriesenhahn
+date: 2019-06-26
+description: |
+  Learn how to create a contributor profile for web.dev.
+---
+
+## Add yourself to the contributors list
+1. Add a new object to [`contributors.json`](https://github.com/GoogleChrome/web.dev/blob/master/src/site/_data/contributors.json) with the following structure. Make sure to choose a unique contributor slug.
+
+    ```json
+    "contributorslug": {
+      "name": {
+        "given": "Jaimie",
+        "family": "Smith"
+      }
+    },
+    ```
+
+1. If you want links to your online accounts to appear in your contributor lockup, add any of these members to your contributor object.
+
+    ```json/5-7
+    "jaimiesmith": {
+      "name": {
+        "given": "Jaimie",
+        "family": "Smith"
+      },
+      "twitter": "jaimiesmith",
+      "github": "jaimiesmith",
+      "glitch": "jaimiesmith"
+    },
+    ```
+
+    <figure class="w-figure  w-figure--center">
+      <img class="w-screenshot" src="./lockup.png" alt="Screenshot of a contributor lockup">
+    </figure>
+
+1. You can also add supplemental info, but it's not currently displayed on web.dev.
+
+    ```json/8-16
+    "jaimiesmith": {
+      "name": {
+        "given": "Jaimie",
+        "family": "Smith"
+      },
+      "twitter": "jaimiesmith",
+      "github": "jaimiesmith",
+      "glitch": "jaimiesmith",
+      "org": {
+        "name": "Google",
+        "unit": "Developer Relations"
+      },
+      "country": "US",
+      "homepage": "https://jaimiesmithis.cool/",
+      "description": {
+        "en": "Jaimie is more than meets the eye"
+      }
+    },
+    ```
+
+## Create a profile image
+1. Edit a photo of yourself so that its dimensions are 192 px x 192 px. The photo should clearly show your face and upper shoulders.
```

Comment:
I think we should start asking folks to drop in two copies.

jane.jpg (96 x 96)
jane@2x.jpg (192 x 192)

I think we'll start using `srcset` soon to swap between the two images if we're on retina displays.